guides/screen-reader-differences.md

```diff
@@ -0,0 +1,126 @@
+
+*This is not the case for elements that are visually hidden using the same CSS techniques as are used to visually hide the "Skip nav" link on every page. It only applies to techniques that hide the content in a straightforward manner such as using `display: none` or `visibility: hidden`.
+
+#### 3. Forms mode
+VoiceOver does not have a forms or application mode. JAWS and NVDA do.
+
+"Since screen readers use many of the keys on the keyboard for quick navigation, trying to enter text in a form field presents an interesting problem. For example, trying to type the letter "h" in browse mode with JAWS would take you out of the form field and navigate to the next heading on the page. Luckily, screen readers have a built-in fix for this issue: Forms mode. In forms mode, most standard screen reader quick keys are deactivated, so that you can fill out forms without accidentally triggering a quick key." [Source](https://webaim.org/articles/jaws/#formsmode)
+
+Read about the specific [JAWS keyboard commands that apply in forms mode](https://webaim.org/articles/jaws/#formsmode) and the [NVDA forms mode](https://webaim.org/articles/nvda/#formsmode).
+
+#### 4. Application mode
+VoiceOver does not have a forms or application mode; JAWS and NVDA do. The application mode is triggered by the use of the `role="application"` ARIA attribute, which disables the default keyboard shortcuts for navigating the page. The developer is then responsible for implementing unique interaction shortcuts with JavaScript.
+
+The W3C recommends developers **not use `role="application"`** if the interface is made up of standard HTML elements such as form elements, links, headings, etc.
+
+"You only want to use `role=application` if the content you’re providing consists of only focusable, interactive controls, and of those, mostly advanced widgets that emulate a real desktop application" ([source](https://www.w3.org/TR/using-aria/#using-application)). This should virtually never be the case for a website that we are developing.
+
+#### 5. VoiceOver on iOS
+
+Swiping and touch screen interactions are the main difference for mobile. Things to watch out for:
+
+- Users must swipe right to jump between focusable elements on the page. This is akin to using the tab key on a keyboard.
+- Swiping left goes back to previous elements, just like using `shift + tab`.
+
+Tabindex, focusable elements, and tab order should be thoroughly tested and verified on small screen interfaces especially to ensure usability with iOS VoiceOver.
+
+
+# Lessons
```

Comment:
You may consider renaming this to something like "Development Recommendations" or "Guidelines" for clarity.